ci-operator/config/maistra/istio/maistra-istio-maistra-2.6.yaml

```diff
@@ -0,0 +1,404 @@
+base_images:
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.5"
```

Comment:
Is this correct? Don't we have to build a 2.6 image first?

Reply:
I think we are using the same builder image, right ?
I don't know any builder change so far.

Reply:
2.6 builder image should align with istio 1.20 image. This is a 2 versions jump in tooling, including golang and libraries.

I'd say having a 2.6 builder image is the first step in bootstrapping 2.6 CI.

Reply:
@yxun Now you can proceed with the changes. While on that, replace `4.12` with `4.14` as well...